calour/experiment.py

```diff
@@ -15,6 +15,8 @@
 import scipy
 import biom
 
+import calour as ca
+
 
```

Comment:
it is probably easier to just revert the changes for the time being. saw you just merged my PR. maybe update based on that?